src/docs/reference/modules/Queries/README.md

```diff
@@ -100,6 +100,10 @@ For Lucene queries with custom object schema, you are limited to elements stored
 
 For SQL queries, you can expose any column where property name is a column alias from the query.
 
+!!! note
+    When returning documents from a SQL query, make sure your query returns a list of document IDs.
```

Comment:
Also add a hint to the checkbox, please.

Reply:
It already has one 👍